src/modules/vigor/components/UserStats.tsx

```diff
@@ -154,19 +153,47 @@ const UserStats: React.FC = () => {
       }
     }
 
-    return <DepositWithdrawTokenRow key={token.contract}>
+    return <TokenButtonRow key={token.contract}>
       <Padding>{commitFormatted}</Padding>
       <Padding hide={commitVal === 0}>{<StyledButton type="button" onClick={handleTokenClick(`WITHDRAW`)}><FormattedMessage id="withdraw" defaultMessage="Withdraw" /></StyledButton>}</Padding>
       <Padding hide={userBalanceVal === 0}>{<StyledButton type="button" onClick={handleTokenClick(`DEPOSIT`)}><FormattedMessage id="deposit" defaultMessage="Deposit" /></StyledButton>}</Padding>
-    </DepositWithdrawTokenRow>
+    </TokenButtonRow>
+  }
+
+  const renderBorrowPayOffRow = (token: TExtendedSymbol) => {
+    const userBalance = userStore.getToken(token.symbol.code)
+    const userDebtFormatted = userStore.userBorrowStats!.debt
+    const userDebtVal = decomposeAsset(userDebtFormatted).amount
+
+    const handleTokenClick = (type: TokenModalTypes) => async () => {
+      // TODO: compute max amount we can borrow here
```

Comment:
This is also where part of #1 would come into play to display how much max VIGOR you can currently borrow

Reply:
👍